themes/src/main/resources/theme/base/admin/resources/js/controllers/clients.js

```diff
@@ -1527,6 +1533,22 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         }
     };
 
+    $scope.changeRequestObjectEncryptionAlg = function() {
+        if ($scope.requestObjectEncryptionAlg === 'any') {
+            $scope.clientEdit.attributes['request.object.encryption.alg'] = null;
+        } else {
+            $scope.clientEdit.attributes['request.object.encryption.alg'] = $scope.requestObjectEncryptionAlg;
+        }
+    };
+
+    $scope.changeRequestObjectEncryptionEncAlg = function() {
```

Comment:
If https://github.com/keycloak/keycloak/pull/8261/files#r672633565 is agreed, it might be better to be `changeRequestObjectEncryptionEnc` .